main.py

```diff
@@ -0,0 +1,52 @@
+import os
+import json
+
+from geolocalizer import Geolocalizer
+
+def hello_world(request):
+    """Responds to any HTTP request.
+    Args:
+        request (flask.Request): HTTP request object.
+    Returns:
+        The response text or any set of values that can be turned into a
+        Response object using
+        `make_response <http://flask.pocoo.org/docs/1.0/api/#flask.Flask.make_response>`.
+    """
+    request_json = request.get_json()
+    if request.args and 'message' in request.args:
+        return request.args.get('message')
+    elif request_json and 'message' in request_json:
+        return request_json['message']
+    else:
+        return f'Hello World!'
+
+def geo_localize_image_map(request, mock=False):
```

Comment:
use geolocalize as one word: geolocalize_image_map.
nit: image vs raster?

Reply:
change to geolocalize_map